config/sunspot.yml

```diff
@@ -0,0 +1,5 @@
+development:
```

Comment:
Can you add the other environments configs here with related ENV variables (ex. database.yml)

Reply:
Added production block and ENVs, didn't think it made sense to have test/staging blocks.